tardis/tardis_portal/download.py

```diff
@@ -276,7 +276,8 @@ def make_tar(self):  # noqa
         for df, name in self.mapped_file_objs:
             fileobj = df.file_object
             self._check('aw')
-            tarinfo = self.gettarinfo(name, name, fileobj)
+            tarinfo = tarfile.TarInfo(name)
```

Comment:
can you use `tarinfo = self.tarinfo(name)` instead, so that it's closer aligned with the super class?